### PR TITLE
update the configuration of `inline` sharding algorithm

### LINKED ISSUE #5234: Update the configuration of `inline` sharding algorithm for Sharding SpringNameSpace
Hi everyone, thanks for your attention.  😉 

Content: Update the configuration of `inline` sharding algorithm for Sharding SpringNameSpace
Difficulty rating: ★☆☆☆☆

### PR DESCRIPTION
**Pull request by strongduanmu**
Fixes #5234 .

Changes proposed in this pull request:
- update the configuration of `inline` sharding algorithm for sharding spring namespace

### CODE REVIEW

File: sharding-spring/sharding-jdbc-spring/sharding-jdbc-spring-namespace/src/test/resources/META-INF/rdb/withNamespaceForMasterSlaveWithStrategyType.xml [hunk 1]
```diff
@@ -19,18 +19,39 @@
...
 
-    <sharding:inline-strategy id="databaseStrategy" sharding-column="user_id" algorithm-expression="dbtbl_${user_id % 2}" />
-    <sharding:inline-strategy id="orderTableStrategy" sharding-column="order_id" algorithm-expression="t_order_${order_id % 4}" />
-    <sharding:inline-strategy id="orderItemTableStrategy" sharding-column="order_id" algorithm-expression="t_order_item_${order_id % 4}" />
-    
+    <spi:sharding-algorithm id="databaseStrategyShardingAlgorithm" type="INLINE" props-ref="databaseProps"/>
+    <spi:sharding-algorithm id="orderTableStrategyShardingAlgorithm" type="INLINE" props-ref="orderTableProps"/>
+    <spi:sharding-algorithm id="orderItemTableStrategyShardingAlgorithm" type="INLINE" props-ref="orderItemTableProps"/>
+
+
```
Comment (tristaZero) [inline sharding-spring/sharding-jdbc-spring/sharding-jdbc-spring-namespace/src/test/resources/META-INF/rdb/withNamespaceForMasterSlaveWithStrategyType.xml:40]: One blank line is enough, please delete either of them.

**Comment by tristaZero** [comment]
@strongduanmu 
Hi, you can always get my point, this PR is good, and just a bit comment needs your attention, thx.

File: sharding-spring/sharding-jdbc-spring/sharding-jdbc-spring-namespace/src/test/resources/META-INF/rdb/withNamespaceForMasterSlaveWithStrategyType.xml (hunk 1 above)
Comment (strongduanmu) [reply sharding-spring/sharding-jdbc-spring/sharding-jdbc-spring-namespace/src/test/resources/META-INF/rdb/withNamespaceForMasterSlaveWithStrategyType.xml:40]: Ok, I will modify it.

**Comment by strongduanmu** [comment]
> @strongduanmu
> Hi, you can always get my point, this PR is good, and just a bit comment needs your attention, thx.

@tristaZero Sorry for this, I will pay attention to these details next time. This problem has been fixed.

**Comment by tristaZero** [comment]
@strongduanmu 
Hi Thanks for rapid feedback. Each line of your beautiful code will present yourself to the whole Apache community. :)

**Pull request merged**